nginx_module.ld-prime

```diff
@@ -3,6 +3,13 @@
 _ngx_modules
```

Comment:
I see `datadog_build_id_nginx_mod` is not being exported, is there a reason for that?

Reply:
addressed in 68a4f0da82bd62d73f2e80627377d4786e102a91